src/style.css

```
@@ -0,0 +1,36 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  display: flex;
+  margin: 0 2px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  height: 16px;
+  width: 16px;
+}
+
+.stars--1 .stars__star:nth-child(1) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--2 .stars__star:nth-of-type(-n+2) {
+  background-image: url(images/star-active.svg);
+}
```

it's better to combine them all in one place

```suggestion
.stars--1 .stars__star:nth-child(1),
.stars--2 .stars__star:nth-of-type(-n+2) {
  background-image: url(images/star-active.svg);
}
```